signup/index.html

```diff
@@ -0,0 +1,50 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>판다마켓 회원가입</title>
+    <link rel="stylesheet" href="style.css" />
+    <link
+      rel="stylesheet"
+      type="text/css"
+      href="https://cdn.jsdelivr.net/gh/orioncactus/pretendard/dist/web/static/pretendard.css"
+    />
+  </head>
+  <body>
+    <a class="logo" href="/"
+      ><img
+        src="/image/Img_logo.png"
+        alt="판다마켓 로고 이미지"
+        width="396px"
+        height="132px"
+    /></a>
+    <div>
+      <form action="">
+        <div>
+          <label for="">이메일</label>
+          <input type="email" />
+          이메일을 입력해주세요
+        </div>
+        <div>
+          <label for="">닉네임</label>
+          <input type="email" />
```

Comment:
```suggestion
<input type="text" />
```
닉네임은 type="text"가 더 적절할 것 같습니다!